package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "version": "3.1.5",
+  "version": "3.2.0",
```

Comment:
Removing an api should be is a breaking change, isn't it?

Reply:
yes, you're right. 4.0

Reply:
4.0 is published